packages/flutter_tools/lib/src/version.dart

```diff
@@ -259,6 +259,13 @@ class FlutterVersion {
         branch: '$_versionCheckRemote/$branch',
         lenient: false,
       );
+    } on VersionCheckError catch(error) {
```

Comment:
nit: Missing a space between `catch` and `(`
```suggestion
    } on VersionCheckError catch (error) {
```